.pre-commit-config.yaml

```diff
@@ -68,6 +60,14 @@ repos:
           - prettier@3.3.3
         args: ["--ignore-path=./superset-frontend/.prettierignore"]
         files: "superset-frontend"
+  - repo: local
```

Comment:
Not directly related to PR, but makes more sense to run prettier first as it's in "fix" more and will often silence double-error-logging between itself and eslint.